assets/js/googlesitekit/modules/datastore/modules.js

```diff
@@ -59,6 +60,7 @@ const moduleDefaults = {
 	order: 10,
 	icon: null,
 	settingsComponent: DefaultModuleSettings,
+	setupComponent: DefaultModuleSetup,
```

Comment:
See above, the default should be `undefined`.

Reply:
I chose to go with `null` for the settings components to indicate that it is intentionally empty as we do elsewhere rather than `undefined` which is more of an initial state or unresolved value. Perhaps we're not consistent with this, but both values should work for our needs. If you would prefer to use `undefined` instead, that's an easy change to make for me.